ccd-definition/ComplexTypes/CaseManagementOrder.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CaseManagementOrder",
+    "ListElementCode": "hearingDate",
+    "FieldType": "Text",
+    "ElementLabel": "Which hearing is this order for?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CaseManagementOrder",
+    "ListElementCode": "hearingDateId",
+    "FieldType": "Text",
+    "ElementLabel": "Hearing Date Id",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "hearingDate=\"DO_NOT_SHOW\""
```

Comment:
could you add a "Comment" field around how this hearingDateId works?

Reply:
done